GoogleTestNuGet/googletest.targets.tt

```diff
@@ -12,18 +12,18 @@
     <Disable-<#= PackageNameDashes #>>true</Disable-<#= PackageNameDashes #>>
   </PropertyGroup>
   <ItemGroup Condition="'$(Disable-<#= PackageNameDashes #>)' == ''">
-    <PropertyPageSchema Include="$(MSBuildThisFileDirectory)\\<#= PackageName #>.propertiesui.xml" />
+    <PropertyPageSchema Include="$(MSBuildThisFileDirectory)\googletest.propertiesui.xml" />
   </ItemGroup>
   <ItemDefinitionGroup Condition="'$(Disable-<#= PackageNameDashes #>)' == ''">
     <Link>
       <AdditionalDependencies Condition="'$(Configuration)' == 'Debug' And '$(Platform)' == 'x64'">$(MSBuildThisFileDirectory)..\..\\<#= PathToBinaries #>\x64\Debug\gtestd.lib;%(AdditionalDependencies)</AdditionalDependencies>
-      <AdditionalDependencies Condition="'$(Configuration)' == 'Debug' And '$(Platform)' == 'x64' And '$(<#= PackageNameDashes #>-Disable-gtest_main)' == ''">$(MSBuildThisFileDirectory)..\..\\<#= PathToBinaries #>\x64\Debug\gtest_maind.lib;%(AdditionalDependencies)</AdditionalDependencies>
+      <AdditionalDependencies Condition="'$(Configuration)' == 'Debug' And '$(Platform)' == 'x64' And '$(GoogleTest-Disable-gtest_main)' == ''">$(MSBuildThisFileDirectory)..\..\\<#= PathToBinaries #>\x64\Debug\gtest_maind.lib;%(AdditionalDependencies)</AdditionalDependencies>
```

Comment:
Wouldn't we want to continue using more "namespaced" `Microsoft-googletest` to prevent possible conflicts?

Reply:
Sure, good call